packages/ingest-mongodb-public/src/sources/mongodb-dot-com/webSources.ts

```diff
@@ -0,0 +1,234 @@
+import puppeteer from "puppeteer";
+import xml2js from "xml2js";
+
+export type RawWebSource = {
```

Comment:
nit: i dislike the word 'raw' here. feels harsh/negative. can you consider different naming convention like `InitialWebSource`,  `UnprocessedWebSource`, etc.